transaction.go

```diff
@@ -26,6 +26,33 @@ const (
 	TransactionStatusUnrecognized           TransactionStatus = "unrecognized"
 )
 
+type ThreeDSecureStatus string
+
+const (
+	ThreeDSecureStatusUnsupportedCard                      ThreeDSecureStatus = "unsupported_card"
+	ThreeDSecureStatusLookupError                          ThreeDSecureStatus = "lookup_error"
+	ThreeDSecureStatusLookupEnrolled                       ThreeDSecureStatus = "lookup_enrolled"
+	ThreeDSecureStatusLookupNotEnrolled                    ThreeDSecureStatus = "lookup_not_enrolled"
+	ThreeDSecureStatusAuthSuccessfulIssuerNotParticipating ThreeDSecureStatus = "authenticate_successful_issuer_not_participating"
+	ThreeDSecureStatusAuthUnavailable                      ThreeDSecureStatus = "authentication_unavailable"
+	ThreeDSecureStatusAuthSignatureVerificationFailed      ThreeDSecureStatus = "authenticate_signature_verification_failed"
+	ThreeDSecureStatusAuthSuccessful                       ThreeDSecureStatus = "authenticate_successful"
+	ThreeDSecureStatusAuthAttemptSuccessful                ThreeDSecureStatus = "authenticate_attempt_successful"
+	ThreeDSecureStatusAuthFailed                           ThreeDSecureStatus = "authenticate_attempt_successful"
+	ThreeDSecureStatusAuthUnableToAuthenticate             ThreeDSecureStatus = "authenticate_unable_to_authenticate"
+	ThreeDSecureStatusAuthError                            ThreeDSecureStatus = "authenticate_error"
+)
+
+type ThreeDSecureEnrollment string
+
+const (
+	ThreeDSecureEnrollementYes            ThreeDSecureEnrollment = "Y"
+	ThreeDSecureEnrollementNo             ThreeDSecureEnrollment = "N"
+	ThreeDSecureEnrollementUnavailable    ThreeDSecureEnrollment = "U"
+	ThreeDSecureEnrollementBypass         ThreeDSecureEnrollment = "B"
+	ThreeDSecureEnrollementRequestFailure ThreeDSecureEnrollment = "E"
```

Comment:
The values are documented [here](https://developers.braintreepayments.com/reference/response/transaction/php#three_d_secure_info.enrolled). I should have put a link in a comment.